packs/data/equipment.db/star-grenade-lesser.json

```diff
@@ -0,0 +1,122 @@
+{
+    "_id": "RsCLCgCgGAn67ry5",
+    "img": "systems/pf2e/icons/default-icons/weapon.svg",
+    "name": "Star Grenade (Lesser)",
+    "system": {
+        "MAP": {
+            "value": ""
+        },
+        "baseItem": "alchemical-bomb",
+        "bonus": {
+            "value": 0
+        },
+        "bonusDamage": {
+            "value": 0
+        },
+        "category": "martial",
+        "containerId": null,
+        "damage": {
+            "damageType": "fire",
+            "dice": 1,
+            "die": "d8",
+            "value": ""
+        },
+        "description": {
+            "value": "<p>These unusual gunpowder bombs, typically marked with a symbol of a four-pointed star on their outer surface, explode outward in the shape of a cross whenever they Strike a target. When you throw a star grenade, arrange two perpendicular @Template[type:line|distance:25]s over the target, both centered on the target and moving straight through the target, not diagonally. Creatures other than the target within these two lines take splash damage (typically 8 squares take splash damage for a Medium creature). Any effect that alters or adjusts the splash area's shape or size, like the alchemist feat Expanded Splash, doesn't apply to star grenades.</p>\n<p>The bomb deals [[/r {1d8}[fire]]]{1d8 fire damage} and [[/r {1}[splash]]]{1 splash Damage}.</p>"
```

Comment:
Can't leave it as a suggestion without github breaking things but `Template[type:line|distance:25]s` should have the label manually added here `Template[type:line|distance:25]{25-foot lines}`.